app/javascript/components/patient/Patient.js

```diff
@@ -68,665 +111,614 @@ class Patient extends React.Component {
+                            {this.props.details.was_in_health_care_facility_with_known_cases_facility_name && (
+                              <span className="risk-val">{this.props.details.was_in_health_care_facility_with_known_cases_facility_name}</span>
+                            )}
+                          </li>
                         )}
                         {this.props.details.laboratory_personnel && (
-                          <Row>
-                            <Col>
-                              <b className="text-danger">LABORATORY PERSONNEL</b>
-                              {this.props.details.laboratory_personnel_facility_name && (
-                                <span>{`: ${this.props.details.laboratory_personnel_facility_name}`}</span>
-                              )}
-                            </Col>
-                          </Row>
+                          <li>
+                            <span className="risk-factor">Laboratory Personnel</span>
+                            {this.props.details.laboratory_personnel_facility_name && (
+                              <span className="risk-val">{this.props.details.laboratory_personnel_facility_name}</span>
+                            )}
+                          </li>
                         )}
                         {this.props.details.healthcare_personnel && (
-                          <Row>
-                            <Col>
-                              <b className="text-danger">HEALTHCARE PERSONNEL</b>
-                              {this.props.details.healthcare_personnel_facility_name && (
-                                <span>{`: ${this.props.details.healthcare_personnel_facility_name}`}</span>
-                              )}
-                            </Col>
-                          </Row>
+                          <li>
+                            <span className="risk-factor">Healthcare Personnel</span>
+                            {this.props.details.healthcare_personnel_facility_name && (
+                              <span className="risk-val">{this.props.details.healthcare_personnel_facility_name}</span>
+                            )}
+                          </li>
                         )}
                         {this.props.details.crew_on_passenger_or_cargo_flight && (
-                          <Row>
-                            <Col>
-                              <b className="text-danger">CREW ON PASSENGER OR CARGO FLIGHT</b>
-                            </Col>
-                          </Row>
+                          <li>
+                            <span className="risk-factor">Crew on Passenger or Cargo Flight</span>
+                          </li>
                         )}
+                      </ul>
+                    )}
+                  </React.Fragment>
+                )}
+                {this.props.details.exposure_notes && !showPotentialExposureInfo && !showRiskFactors && (
```

Comment:
Why do we need to duplicate this code when we have this above in the case where `showPotentialExposureInfo || showRiskFactors`? Seems like we should just have one check for `this.props.details.exposure_notes` for displaying this.

Reply:
The reason for this is there are two possible sections where the exposure notes can be shown.  If there is no exposure info and just notes, we display it in the PEI section (see case 16 of Mike's mockups).  Otherwise it is shown as it's own section (all other cases).  The additional check here is to be sure the same notes section is not shown twice